TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:37.653, 1340913677438, datacollector.dll, DataCollectorMain.Run: Starting data collector run with args: --port,56881,--parentprocessid,8280,--diag,C:\Users\dzene\Source\Repos\VVS-Z2\Hypo Banka\Unit Testovi\bin\Debug\fine-code-coverage\_outputfolder\diagnostics.datacollector.20-11-28_22-19-37_31662_4.log,--tracelevel,4
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:37.666, 1340913738116, datacollector.dll, DataCollector: Monitoring parent process with id: '8280'
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:37.762, 1340914695673, datacollector.dll, Trying to connect to server on socket : 127.0.0.1:56881 
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:37.786, 1340914940056, datacollector.dll, SocketCommunicationManager : SetupClientAsync : Attempting to connect to the server.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:37.809, 1340915168488, datacollector.dll, Connected to the server successfully 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:37.809, 1340915171216, datacollector.dll, Using the buffer size of 16384 bytes
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:37.957, 1340916648821, datacollector.dll, DataCollector: Start Request Processing.
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:37.964, 1340916715300, datacollector.dll, DataCollectionRequestHandler.ProcessRequests : Datacollector received message: (DataCollection.BeforeTestRunStart) -> {
  "SettingsXml": "<RunSettings><RunConfiguration><ResultsDirectory>C:\\Users\\dzene\\Source\\Repos\\VVS-Z2\\Hypo Banka\\Unit Testovi\\bin\\Debug\\fine-code-coverage\\_outputfolder</ResultsDirectory><TargetPlatform>X64</TargetPlatform><TargetFrameworkVersion>.NETCoreApp,Version=v3.1</TargetFrameworkVersion><DesignMode>False</DesignMode><CollectSourceInformation>False</CollectSourceInformation><TestAdaptersPaths>C:\\Program Files\\dotnet\\sdk\\3.1.201\\Extensions</TestAdaptersPaths></RunConfiguration><LoggerRunSettings><Loggers><Logger friendlyName=\"blame\" enabled=\"True\" /><Logger friendlyName=\"Console\" uri=\"logger://microsoft/TestPlatform/ConsoleLogger/v1\" assemblyQualifiedName=\"Microsoft.VisualStudio.TestPlatform.CommandLine.Internal.ConsoleLogger, vstest.console, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a\" codeBase=\"C:\\Program Files\\dotnet\\sdk\\3.1.201\\vstest.console.dll\" enabled=\"True\" /></Loggers></LoggerRunSettings><DataCollectionRunSettings><DataCollectors><DataCollector friendlyName=\"blame\" enabled=\"True\"><Configuration><ResultsDirectory>C:\\Users\\dzene\\Source\\Repos\\VVS-Z2\\Hypo Banka\\Unit Testovi\\bin\\Debug\\fine-code-coverage\\_outputfolder</ResultsDirectory></Configuration></DataCollector></DataCollectors></DataCollectionRunSettings></RunSettings>",
  "Sources": [
    "C:\\Users\\dzene\\Source\\Repos\\VVS-Z2\\Hypo Banka\\Unit Testovi\\bin\\Debug\\fine-code-coverage\\Unit Testovi.dll"
  ]
}
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.048, 1340917555005, datacollector.dll, TestPluginCache: Update extensions started. Skip filter = False
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.049, 1340917567983, datacollector.dll, TestPluginCache: Using directories for assembly resolution 'C:\Program Files\dotnet\sdk\3.1.201\Extensions'.
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.049, 1340917569730, datacollector.dll, TestPluginCache: Updated the available extensions to 'C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.BlameDataCollector.dll,C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.EventLogCollector.dll'.
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.063, 1340917708474, datacollector.dll, TestPluginCache.DiscoverTestExtensions: finding test extensions in assemblies endswith: Collector.dll TPluginInfo: Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectorConfig TExtension: Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollector
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.063, 1340917712291, datacollector.dll, TestPluginCache.GetExtensionPaths: Filtered extension paths: C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.BlameDataCollector.dll
C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.EventLogCollector.dll
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.064, 1340917714938, datacollector.dll, TestPluginCache.GetExtensionPaths: Added default extension paths: 
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.064, 1340917717119, datacollector.dll, TestPluginCache.GetExtensionPaths: Added unfilterableExtensionPaths: 
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:38.066, 1340917735808, datacollector.dll, AssemblyResolver.ctor: Creating AssemblyResolver with searchDirectories C:\Program Files\dotnet\sdk\3.1.201\Extensions,C:\Program Files\dotnet\sdk\3.1.201
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.069, 1340917765872, datacollector.dll, TestPluginCache.DiscoverTestExtensions: Discovering the extensions using extension path.
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.069, 1340917770020, datacollector.dll, TestPluginCache.GetExtensionPaths: Filtered extension paths: C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.BlameDataCollector.dll
C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.EventLogCollector.dll
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.070, 1340917772416, datacollector.dll, TestPluginCache.GetExtensionPaths: Added default extension paths: 
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.070, 1340917774303, datacollector.dll, TestPluginCache.GetExtensionPaths: Added unfilterableExtensionPaths: 
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.070, 1340917776747, datacollector.dll, TestPluginCache.DiscoverTestExtensions: Discovering the extensions using allExtensionPaths: C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.BlameDataCollector.dll
C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.EventLogCollector.dll
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:38.070, 1340917780503, datacollector.dll, AssemblyResolver.AddSearchDirectories: Adding more searchDirectories C:\Program Files\dotnet\sdk\3.1.201\Extensions,C:\Program Files\dotnet\sdk\3.1.201
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:38.071, 1340917783094, datacollector.dll, AssemblyResolver.AddSearchDirectories: Adding more searchDirectories C:\Program Files\dotnet\sdk\3.1.201\Extensions,C:\Program Files\dotnet\sdk\3.1.201
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:38.072, 1340917794797, datacollector.dll, AssemblyResolver.OnResolve: Microsoft.TestPlatform.Extensions.BlameDataCollector: Resolving assembly.
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:38.075, 1340917823255, datacollector.dll, AssemblyResolver.OnResolve: Resolved assembly: Microsoft.TestPlatform.Extensions.BlameDataCollector, from path: C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.BlameDataCollector.dll
TpTrace Error: 0 : 6496, 1, 2020/11/28, 22:19:38.076, 1340917837784, datacollector.dll, The type "Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameCollector" defined in "Microsoft.TestPlatform.Extensions.BlameDataCollector.dll" does not have ExtensionUri attibute.
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:38.077, 1340917849075, datacollector.dll, AssemblyResolver.OnResolve: Microsoft.TestPlatform.Extensions.EventLogCollector: Resolving assembly.
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:38.079, 1340917868985, datacollector.dll, AssemblyResolver.OnResolve: Resolved assembly: Microsoft.TestPlatform.Extensions.EventLogCollector, from path: C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.EventLogCollector.dll
TpTrace Error: 0 : 6496, 1, 2020/11/28, 22:19:38.083, 1340917906748, datacollector.dll, The type "Microsoft.TestPlatform.Extensions.EventLogCollector.EventLogDataCollector" defined in "Microsoft.TestPlatform.Extensions.EventLogCollector.dll" does not have ExtensionUri attibute.
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.084, 1340917912958, datacollector.dll, TestPluginCache: Discovered the extensions using extension path 'C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.BlameDataCollector.dll,C:\Program Files\dotnet\sdk\3.1.201\Extensions\Microsoft.TestPlatform.Extensions.EventLogCollector.dll'.
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.084, 1340917914529, datacollector.dll, TestPluginCache: Discoverers are ''.
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.084, 1340917915779, datacollector.dll, TestPluginCache: Executors are ''.
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.084, 1340917916891, datacollector.dll, TestPluginCache: Setting providers are ''.
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:38.084, 1340917917970, datacollector.dll, TestPluginCache: Loggers are ''.
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:38.086, 1340917940205, datacollector.dll, TestPluginManager.CreateTestExtension: Attempting to load test extension: Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameCollector
TpTrace Error: 0 : 6496, 1, 2020/11/28, 22:19:38.087, 1340917942948, datacollector.dll, The type "Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameCollector" defined in "Microsoft.TestPlatform.Extensions.BlameDataCollector.dll" does not have ExtensionUri attibute.
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:38.096, 1340918035682, datacollector.dll, Listening on Endpoint : 127.0.0.1:56883
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:38.159, 1340918669584, datacollector.dll, DataCollectionRequestHandler.ProcessRequests : DataCollection started.
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:38.224, 1340919314507, datacollector.dll, DataCollectionRequestHandler.ProcessRequests : Datacollector received message: (DataCollection.TestHostLaunched) -> {
  "ProcessId": 7580
}
TpTrace Information: 0 : 6496, 6, 2020/11/28, 22:19:38.595, 1340923023844, datacollector.dll, Using the buffer size of 16384 bytes
TpTrace Information: 0 : 6496, 6, 2020/11/28, 22:19:38.595, 1340923027598, datacollector.dll, Accepted Client request and set the flag
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.567, 1340932751657, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.630, 1340933376517, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.631, 1340933388502, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.840, 1340935483064, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.869, 1340935764889, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:39.907, 1340936143737, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.914, 1340936216268, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.931, 1340936386036, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.932, 1340936399130, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.932, 1340936402173, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.936, 1340936437811, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.937, 1340936449875, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:39.938, 1340936455389, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.938, 1340936458816, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.940, 1340936480362, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.941, 1340936486766, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.941, 1340936489932, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.944, 1340936519895, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.950, 1340936579232, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:39.951, 1340936583447, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.951, 1340936586642, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.953, 1340936610493, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.954, 1340936616126, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:39.954, 1340936618690, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.061, 1340937688548, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.062, 1340937695874, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.062, 1340937698960, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.062, 1340937701689, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.064, 1340937716459, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.064, 1340937721886, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.065, 1340937725057, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.070, 1340937776585, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.071, 1340937782755, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.071, 1340937786116, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.071, 1340937789077, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.073, 1340937805604, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.073, 1340937811442, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.074, 1340937813755, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.076, 1340937838089, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.077, 1340937843562, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.077, 1340937846648, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.077, 1340937849550, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.079, 1340937864906, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.079, 1340937870150, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.079, 1340937872393, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.082, 1340937900663, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.083, 1340937906103, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.083, 1340937909039, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.083, 1340937911386, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.085, 1340937924640, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.085, 1340937929985, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.086, 1340937932582, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.089, 1340937964645, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.089, 1340937970456, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.090, 1340937973525, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.090, 1340937976076, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.091, 1340937989459, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.092, 1340937994480, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.092, 1340937996896, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.094, 1340938018346, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.095, 1340938024064, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.095, 1340938027017, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.095, 1340938029455, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.229, 1340939366753, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.232, 1340939399125, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.233, 1340939411454, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.238, 1340939454442, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.240, 1340939481795, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.242, 1340939494523, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.243, 1340939507366, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.245, 1340939532366, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.248, 1340939556562, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.249, 1340939565996, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.255, 1340939632672, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.257, 1340939652164, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.259, 1340939663798, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.260, 1340939672768, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.262, 1340939696596, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.264, 1340939715960, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.265, 1340939726043, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.275, 1340939827921, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.277, 1340939844573, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.278, 1340939854863, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.279, 1340939863578, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.281, 1340939887324, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.283, 1340939905065, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.283, 1340939912258, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.296, 1340940034117, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.298, 1340940057358, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.299, 1340940065187, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.300, 1340940072718, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.302, 1340940095494, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.303, 1340940104043, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.303, 1340940108060, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.310, 1340940181470, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.311, 1340940190343, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.312, 1340940194550, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.312, 1340940197609, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.314, 1340940219571, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.315, 1340940228497, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.315, 1340940232112, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.319, 1340940272191, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.320, 1340940281786, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.321, 1340940286880, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.321, 1340940291166, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.323, 1340940311753, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.324, 1340940320264, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.325, 1340940324320, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.328, 1340940354134, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.328, 1340940361474, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.329, 1340940366113, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.329, 1340940370447, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.332, 1340940392885, datacollector.dll, DataCollectionTestCaseEventHandler: Test case starting.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.332, 1340940401360, datacollector.dll, Blame Collector : Test Case Start
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.333, 1340940405660, datacollector.dll, DataCollectionTestCaseEventHandler: Test case started.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.336, 1340940434158, datacollector.dll, DataCollectionTestCaseEventHandler : Test case completing.
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.336, 1340940441810, datacollector.dll, Blame Collector : Test Case End
TpTrace Error: 0 : 6496, 4, 2020/11/28, 22:19:40.337, 1340940445777, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.337, 1340940449225, datacollector.dll, DataCollectionTestCaseEventHandler: Test case completed
TpTrace Information: 0 : 6496, 4, 2020/11/28, 22:19:40.346, 1340940534349, datacollector.dll, DataCollectionTestCaseEventHandler: Test session ended
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:40.527, 1340942348923, datacollector.dll, DataCollectionRequestHandler.ProcessRequests : Datacollector received message: (DataCollection.AfterTestRunEnd) -> false
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:40.528, 1340942360326, datacollector.dll, Blame Collector : Session End
TpTrace Error: 0 : 6496, 1, 2020/11/28, 22:19:40.529, 1340942365526, datacollector.dll, DataCollectionAttachmentManager.GetAttachments: Fail to get attachments: System.Collections.Generic.KeyNotFoundException: The given key 'Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionContext' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionAttachmentManager.GetAttachments(DataCollectionContext dataCollectionContext) 
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:40.529, 1340942367333, datacollector.dll, DataCollectionManager.CleanupPlugins: CleanupPlugins called
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:40.529, 1340942368677, datacollector.dll, DataCollectionManager.CleanupPlugins: Cleaning up 1 plugins
TpTrace Verbose: 0 : 6496, 1, 2020/11/28, 22:19:40.529, 1340942371123, datacollector.dll, dataCollectorInfo.DisposeDataCollector: calling Dispose() on Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameCollector
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:40.530, 1340942372972, datacollector.dll, DataCollectionManager.CleanupPlugins: CleanupPlugins finished
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:40.530, 1340942377186, datacollector.dll, DataCollectionRequestHandler.ProcessRequests : Session End message received from server. Closing the connection.
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:40.530, 1340942381444, datacollector.dll, Closing the connection !
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:40.531, 1340942383581, datacollector.dll, DataCollectionRequestHandler.ProcessRequests : DataCollection completed
TpTrace Information: 0 : 6496, 1, 2020/11/28, 22:19:40.531, 1340942385466, datacollector.dll, Program.Main: exiting datacollector process.
